paper/paper.md

```diff
@@ -21,158 +21,63 @@ authors:
+For frequently changing datasets, this may not be sufficient.
+After exploring, analysing and documenting it once, an understanding needs to be continously verified to be up-to-date, and adjusted when it is not.
 To address these problems, we developed `datapact`, a Python library that provides an ergonomic API for running assertions on DataFrames.
 
+`datapact` is available under MIT license at https://github.com/skn0tt/datapact.
+
 # Statement of need
 
 When working with continually changing datasets, e.g. occupancy statistics for intensive care units during a pandemic, dataset shift (@moreno2012unifying) will occur frequently.
 Sometimes because of human errors in data entry, sometimes because of bugs, errors or outages in the data delivery, and sometimes because there are changes in the underlying data.
-If these changes stay unnoticed, they can make their way into subtle bugs, faulty visualisations and ultimately lead to mislead decisions.
-The faster they're noticed, the faster analysis scripts, dashboards and data pipelines can be updated. TODO: citatation needed
+If these changes stay unnoticed, they can make their way into subtle bugs, faulty visualisations and ultimately lead to suboptimal decisions.
+The faster they get noticed, the faster analysis scripts, dashboards and data pipelines can be updated.
 
 Without dedicated systems to detect them, changes can often go unnoticed for months.
 `datapact` enables such a system to be built easily, so changes are noticed as the data comes in.
-It is a Python library for expressing + verifying characteristics on DataFrames, and can be embedded right within the Python scripts and Jupyter notebooks that are already being used for analysis.
-Different tests are needed, both simple sanity checks ("the ICU patient's age should be between 0 and 150") and more complex characteristics ("the age should be normal distributed").
-Specialized tests should be easily addable via custom code.
+It is a Python library for expressing and verifying characteristics on DataFrames, and can be embedded right within the Python scripts and Jupyter notebooks that are already being used for analysis.
+Different tests are needed, both simple sanity checks ("the patient's age should be between 0 and 125") and more complex characteristics ("the age should be normally distributed").
+More specialized tests can easily be added via custom code.
 
-Great Expectations (@Gong_Great_Expectations) is an existing framework that addresses these issues via Pipeline Tests. It can be hard to learn and hard to set up, especially for non-engineering-heavy organisations.
+Existing projects addressing this need (e.g. Great Expectations (@Gong_Great_Expectations)) come with comparatively high entrance hurdles that are hard to overcome, particularly for non engineering heavy organisations.
```

Comment:
```suggestion
Existing projects addressing this need, e.g., Great Expectations (@Gong_Great_Expectations) come with comparatively high entrance hurdles that are hard to overcome, particularly for non engineering heavy organisations.
```

Reply:
```suggestion
Existing projects addressing this need, e.g. Great Expectations (@Gong_Great_Expectations), come with comparatively high entrance hurdles that are hard to overcome, particularly for non engineering heavy organisations.
```

this looks better to me, and according to https://linguaholic.com/linguablog/comma-after-e-g/ it's okay to omit the comma after `e.g.`

Reply:
Sounds good